addons/website_event/models/event_registration.py

```diff
@@ -9,3 +9,6 @@ class EventRegistration(models.Model):
     _inherit = ['event.registration']
 
     visitor_id = fields.Many2one('website.visitor', string='Visitor', ondelete='set null')
+
+    def _get_allowed_fields(self):
```

Comment:
Could probable be better named :D

Reply:
_get_website_registration_allowed_fields ?

Reply:
Allez. J'ai pas mieux en stock.